core/src/main/java/org/axonframework/eventhandling/tokenstore/TokenStore.java

```diff
@@ -50,6 +50,29 @@ default void initializeTokenSegments(String processorName, int segmentCount) thr
         }
     }
 
+    /**
+     * Initializes the given {@code segmentCount} number of segments for the given {@code processorName} to track its
+     * tokens. This method should only be invoked when no tokens have been stored for the given processor, yet.
+     * <p>
+     * This method will store {@code initialToken} for all segments as starting point for processor, but no claim them.
```

Comment:
I assume you've got a typo here. `..., but not claim them.` i.o. `..., but no claim them.`